ERCS/erc-4824.md

```diff
@@ -53,9 +53,7 @@ The DAO JSON-LD Schema mentioned above:
 }
 ```
 
-A DAO MAY inherit the above interface above or it MAY create an external registration contract that is compliant with this EIP. If a DAO creates an external registration contract, the registration contract MUST store the DAO’s primary address. 
-
-If the DAO inherits the above interface, it SHOULD define a method for updating daoURI. If the DAO uses an external registration contract, the registration contract SHOULD contain some access control logic to enable efficient updating for daoURI.
+A DAO MAY inherit the interface above or it MAY create an external registration contract that is compliant with this EIP. If a DAO creates an external registration contract, the registration contract MUST store the DAO’s primary address.
 
 ```solidity
```

Comment:
I _think_ this contract should be in the reference implementation section. It doesn't sound like the internals of this contract need to be exactly this, and any compatible implementation would work just as well. I'd recommend making this an interface, and moving the implementation to the reference implementation section.

---

Also your contract name seems to be a victim of find/replace.

Reply:
moved the contract code to assets in the new commit, and fixed the find/replace error with the double title